http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -211,6 +211,32 @@ class ProxyService(
       r <- proxyInternal(hostContext, request)
     } yield r
 
+    res.onError {
+      case e =>
+        IO(
+          logger.warn(
+            s"${ctx.traceId} | proxy request failed for ${userInfo.userEmail.value} ${googleProject.value} ${runtimeName.asString}",
```

Comment:
May be easier to parse the log message if we specify what `${...}` values are...something like

```suggestion
            s"${ctx.traceId} | proxy request failed for user/project/runtime ${userInfo.userEmail.value}/${googleProject.value}/${runtimeName.asString}"
```